Add list of all types and powers to sort ingridients

diana-dudak opened the pull request:
Hi, i really like your tool, but i felt like it was missing the list of all types and powers to sort ingridients. (Searches only sorted recepies.) So i added one.
It looks like this and is visible when Simple mode is turned OFF:
![preview](https://user-images.githubusercontent.com/49321699/206529952-06253687-821e-41a1-a7cd-522915952913.png)

I hope you like it ^_^

cecilbowen commented:
This functionality is already possible by clicking the flavors/powers/types in the cards.
![image](https://user-images.githubusercontent.com/45740102/206606742-25eed6e3-57a8-4ab4-ae71-74b89504c736.png)

But I'm not opposed to supplementing it with the selectors you've added at the top so that users can use either one.  If you add the flavors up there as well, make sure that the current card selectors and your selectors work interchangeably and maybe remove the headers to reduce screen space, then I'd say it's be good to go.

diana-dudak commented:
Yes, i used your cards functionality there. My issue with cards is that i can't see the whole list of options and need to randomly select ingridients until i find correct type or power to sort with.
I updated PR, added flavors and removed headings. It looks like this: 
![preview](https://user-images.githubusercontent.com/49321699/206685329-e83c775c-4a86-4445-aadd-4420af8a8743.png)
